packages/runner-ct/src/app/useScreenshotHandler.tsx

```diff
@@ -25,6 +25,10 @@ export function useScreenshotHandler ({ eventManager, state, splitPaneRef }: {
 
     splitPaneRef.current.splitPane.firstElementChild.classList.remove('d-none')
     splitPaneRef.current.splitPane.querySelector('[role="presentation"]').classList.remove('d-none')
+
+    const iframe = document.querySelector<HTMLIFrameElement>('.aut-iframe')
+
+    iframe.classList.remove('aut-iframe-screenshotting')
```

Comment:
This is pretty dirty, but unfortunately the `<iframe>` is not controlled by React. It's initialized [here](https://github.com/cypress-io/cypress/blob/8700ecf39b09c803eaeabcc22aa502aa11612334/packages/runner-ct/src/iframe/iframes.jsx#L60), and actually created [here](https://github.com/cypress-io/cypress/blob/8700ecf39b09c803eaeabcc22aa502aa11612334/packages/runner-ct/src/iframe/aut-iframe.js#L17) using jquery, so I don't any better way to do this. We need some specific styling to be applied during screenshotting to make everything look correct.

Reply:
This might be OK temporarily, but the iframe code lives entirely inside `runner-ct`, so we can refactor it to be rendered via React.

Reply:
Agreed

Reply:
The real value of this PR is we now have some Percy screenshots that are correct, so we can safely refactor.